src/box/txn_limbo.c

```diff
@@ -208,7 +212,16 @@ txn_limbo_abort(struct txn_limbo *limbo, struct txn_limbo_entry *entry)
 	 * rollback in the reversed order. Rolled back transaction
 	 * is always the last.
 	 */
```

Comment:
The comment above was for `txn_limbo_pop` a few lines below. Lets keep the comment and that code line together.

Reply:
Fixed.